NOTICE

```diff
@@ -3,9 +3,15 @@ Coulter, Inc. It is released under the terms of the MIT license.
 
 This product also includes separately licensed:
 
+* Chez Scheme by Cisco Systems, Inc.
+
 * jQuery by the jQuery Foundation
 
+* libuv
```

Comment:
Most other entries are of the form "xyz by organization". Is there anything useful for libuv that we should add?

Reply:
I couldn't find an organization behind it. Since it's so well known, I think it's OK as is.